sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -418,4 +431,19 @@ class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleto
       assert(sql("SHOW PARTITIONS part_datasrc").count() == 3)
     }
   }
+
+  /**
+   * Run a function with a copy of the input file. Use this for tests that use "LOAD DATA"
+   * (instead of "LOAD DATA LOCAL") since, according to Hive's semantics, files are moved
```

Comment:
then can we test `LOAD DATA` and `LOAD DATA LOCAL` separately? We can add comments to explain the semantic difference between them and why we need to copy the file